stock_barcodes/wizard/stock_barcodes_read_picking.py

```diff
@@ -40,6 +43,25 @@ class WizStockBarcodesReadPicking(models.TransientModel):
         string='Confirmed moves',
     )
 
+    @api.depends(
+        "picking_id", "barcode", "picking_id.move_lines.move_line_ids.qty_done"
+    )
+    def _compute_pending_move(self):
+        for record in self:
```

Comment:
Compute field only if user has the security group for performance.

Reply:
Done :+1: